packages/react-console/src/components/AccessConsoles/AccessConsoles.tsx

```diff
@@ -100,6 +100,7 @@ export const AccessConsoles: React.FunctionComponent<AccessConsolesProps> = ({
       {React.Children.toArray(children).length > 1 && (
         <div className="pf-c-console__actions">
           <Select
+            aria-label="Select Console Input"
```

Comment:
`aria-label={textSelectConsoleType}`

Reply:
Ah indeed, I am not even sure if that's what broke the a11y tests though, just a guess.

Reply:
It was not that